src/openforms/formio/service.py

```diff
@@ -105,3 +107,9 @@ def get_dynamic_configuration(
     # as checkboxes/dropdowns/radios/...
     inject_prefill(config_wrapper, submission)
     return config_wrapper
+
+
+def build_serializer(
+    components: list[Component], _register: ComponentRegistry | None = None, **kwargs
+):
+    return _build_serializer(components, register=_register or register, **kwargs)
```

Comment:
If this is supposed to be the public function, would be great to have the docstring here as well